Cargo.toml

```diff
@@ -1,14 +1,14 @@
 [package]
 name = "cargo-ft"
-version = "0.1.2"
+version = "0.1.3"
```

Comment:
You don't need to change this, there is a Github action automating the process for release